RELEASES.md

```diff
@@ -0,0 +1,36 @@
+OpenZFS uses the MAJOR.MINOR.PATCH versioning scheme described here:
+
+  * MAJOR - Incremented at the discretion of the OpenZFS developers to indicate
+    a particularly noteworthy feature or change. An increase in MAJOR number
+    does not indicate any incompatible on-disk format change. The ability
+    to import a ZFS pool is controlled by the feature flags enabled on the
+    pool and the feature flags supported by the installed OpenZFS version.
+    Increasing the MAJOR version is expected to be an infrequent occurrence.
+
+  * MINOR - Incremented to indicate new functionality such as a new feature
+    flag, pool/dataset property, zfs/zpool sub-command, new user/kernel
+    interface, etc. MINOR releases may introduce incompatible changes to the
+    user space library APIs (libzfs.so, libzfs_core.so, etc).  Existing
```

Comment:
```suggestion
    user space library APIs (e.g. libzfs.so).  Existing
```

We want libzfs_core to never have incompatible changes (at least as a goal if not hard requirement), so let's not list it here.

Reply:
We want it to always be backwards compatible which I think is a little bit different.  Regardless, I've removed it to make this more clear.